src/components/privacy/index.tsx

```diff
@@ -0,0 +1,886 @@
+import Link from 'next/link'
+import MUILink from '@mui/material/Link'
+import css from './styles.module.css'
+
+const SafePrivacyPolicy = () => {
+  return (
+    <div>
+      <h1>Privacy Policy</h1>
+      <p>Last updated on March&nbsp;2023.</p>
+      <p>
+        Your privacy is important to us. It is our policy to respect your privacy and comply with any applicable law and
+        regulation regarding any personal information we may collect about you, including across our website,{' '}
+        <a href="https://app.safe.global">https://app.safe.global</a>
```

Comment:
There are a few anchor elements on this page. Should they be `MUILink`?

Reply:
Changed it